client/.prettierignore

```diff
@@ -0,0 +1,2 @@
+*.tsx
+*.ts
```

Comment:
Same concerns as [PR #44](https://github.com/hatchways/team-runtime-terror/pull/44#discussion_r687001566)

Reply:
actually, I like to use the ESLint auto fixer at the end to correct all my linter errors, rather than prettier constantly showing up with all the errors.